packages/client/internals/PresenterPrint.vue

```diff
@@ -25,7 +25,7 @@ const slidesWithNote = computed(() => rawRoutes
         </div>
       </div>
 
-      <div v-for="(slide, index) of slidesWithNote" :key="index" class="flex flex-col gap-4">
+      <div v-for="(slide, index) of slidesWithNote" :key="index" class="flex flex-col gap-4 break-inside-avoid-page">
```

Comment:
Avoid to break a single slide block. It's more usable if the slide comments are all together in a single printed page